.vscode/settings.json

```diff
@@ -27,4 +27,4 @@
     "files.encoding": "utf8bom",
     "files.insertFinalNewline": true
   }
-}
\ No newline at end of file
+}
```

Comment:
Seems like an unintended change ;)

Reply:
For some reason if I remove that my build fails complaining about prettier checks